.github/workflows/main.yml

```diff
@@ -0,0 +1,348 @@
+name: CI
+
+on:
+  # TODO: add nightly schedule.
```

Comment:
So no nightly release right now?

Reply:
Yea, although I could probably get that done by the end of the day